### PR TITLE
Update Face to 2.7.0-preview.3

### PR DESCRIPTION
**Pull request by msyache**
Generated changes from 
https://github.com/Azure/azure-rest-api-specs/commit/049415ccedf05285a13da035877ce3280e629634

### CODE REVIEW

File: sdk/cognitiveservices/Vision.Face/src/Generated/FaceOperations.cs [hunk 1]
```diff
@@ -970,7 +970,8 @@ public async Task<HttpOperationResponse<VerifyResult>> VerifyFaceToFaceWithHttpM
         /// </param>
         /// <param name='returnFaceId'>
         /// A value indicating whether the operation should return faceIds of detected
-        /// faces.
+        /// faces. Detecting with this value set to true requires additional
+        /// permissions.
```
Comment (dipidoo) [inline sdk/cognitiveservices/Vision.Face/src/Generated/FaceOperations.cs:974]: "permissions" here is most likely going to be misinterpreted as "admin permission." Perhaps create a fwlink to the gating explanation page and put it here?

File: sdk/cognitiveservices/Vision.Face/tests/SessionRecords/FaceDetectionTests/FaceDetection03WithAttributes.json [hunk 2]
```diff
@@ -23,168 +23,35 @@
...
...r\n        \"x\": 632.3,\r\n        \"y\": 228.4\r\n      },\r\n      \"noseRootLeft\": {\r\n        \"x\": 598.8,\r\n        \"y\": 240.0\r\n      },\r\n      \"noseRootRight\": {\r\n        \"x\": 614.0,\r\n        \"y\": 235.1\r\n      },\r\n      \"noseLeftAlarTop\": {\r\n        \"x\": 603.1,\r\n        \"y\": 260.7\r\n      },\r\n      \"noseRightAlarTop\": {\r\n        \"x\": 626.3,\r\n        \"y\": 251.4\r\n      },\r\n      \"noseLeftAlarOutTip\": {\r\n        \"x\": 601.5,\r\n        \"y\": 274.1\r\n      },\r\n      \"noseRightAlarOutTip\": {\r\n        \"x\": 634.1,\r\n        \"y\": 262.3\r\n      },\r\n      \"upperLipTop\": {\r\n        \"x\": 627.8,\r\n        \"y\": 288.2\r\n      },\r\n      \"upperLipBottom\": {\r\n        \"x\": 628.2,\r\n        \"y\": 293.5\r\n      },\r\n      \"underLipTop\": {\r\n        \"x\": 629.0,\r\n        \"y\": 296.4\r\n      },\r\n      \"underLipBottom\": {\r\n        \"x\": 630.9,\r\n        \"y\": 303.6\r\n      }\r\n    },\r\n    \"faceAttributes\": {\r\n      \"mask\": {\r\n        \"type\": \"noMask\",\r\n        \"noseAndMouthCovered\": false\r\n      }\r\n    },\r\n    \"recognitionModel\": \"recognition_04\"\r\n  }\r\n]",
```
Comment (dipidoo) [inline sdk/cognitiveservices/Vision.Face/tests/SessionRecords/FaceDetectionTests/FaceDetection03WithAttributes.json:54]: [nit] can this be formatted as before?

File: sdk/cognitiveservices/Vision.Face/src/Generated/Models/Snapshot.cs [hunk 3]
```diff
@@ -18,7 +18,6 @@ namespace Microsoft.Azure.CognitiveServices.Vision.Face.Models
 
     /// <summary>
     /// Snapshot object.
-    /// </summary>
```
Comment (dipidoo) [inline sdk/cognitiveservices/Vision.Face/src/Generated/Models/Snapshot.cs:21]: is this auto-generated? how did the </summary> tag become lost?

File: sdk/cognitiveservices/Vision.Face/tests/SessionRecords/FaceDetectionTests/FaceDetection03WithAttributes.json (hunk 2 above)
Comment (msyache) [reply sdk/cognitiveservices/Vision.Face/tests/SessionRecords/FaceDetectionTests/FaceDetection03WithAttributes.json:54]: As this is generated by SessionRecorder and is only used to feed the mock http calls, I think we can keep it in the original generated form.

**Review by dipidoo — APPROVED** [review]

**Pull request merged**